src/cpu/core_dynrec/risc_ppc64le.h

```diff
@@ -641,30 +642,30 @@ static void gen_fill_branch(DRC_PTR_SIZE_IM data)
 // conditional jump if register is nonzero
 // for isdword==true the 32bit of the register are tested
 // for isdword==false the lowest 8bit of the register are tested
-static uint64_t gen_create_branch_long_nonzero(HostReg reg, bool dword)
+static const uint8_t* gen_create_branch_long_nonzero(HostReg reg, bool dword)
 {
 	if (!dword)
 		IMM_OP(28,reg,HOST_R0,0xFF); // andi. r0,reg,0xFF
 	else
 		IMM_OP(11, 0, reg, 0);       // cmpwi cr0, reg, 0
 
 	IMM_OP(16, 0x04, 2, 0); // bne
-	return ((uint64_t)cache.pos-4);
+	return ((const uint8_t*)(cache.pos-4));
 }
 
 // compare 32bit-register against zero and jump if value less/equal than zero
-static uint64_t gen_create_branch_long_leqzero(HostReg reg)
+static const uint8_t* gen_create_branch_long_leqzero(HostReg reg)
 {
 	IMM_OP(11, 0, reg, 0); // cmpwi cr0, reg, 0
 
 	IMM_OP(16, 0x04, 1, 0); // ble
-	return ((uint64_t)cache.pos-4);
+	return ((const uint8_t*)(cache.pos-4));
 }
 
 // calculate long relative offset and fill it into the location pointed to by data
-static void gen_fill_branch_long(uint64_t data)
+static void gen_fill_branch_long(const uint8_t* udata)
 {
-	return gen_fill_branch((DRC_PTR_SIZE_IM)data);
+	return gen_fill_branch(udata);
```

Comment:
(minor: please `data`; the coding style doesn't use type-or-signedness prefix-notation in variable names, like uname or sname).

Reply:
done